.buildkite/scripts/dra_release.sh

```diff
@@ -4,14 +4,15 @@ set -euo pipefail
 
 source .buildkite/scripts/common.sh
 
-VERSION="$(awk '/const DefaultVersion/{print $NF}' version/version.go | tr -d '"')"
 FOLDER_PATH="build/distributions"
 BASE_DIR="${WORKSPACE}/${FOLDER_PATH}"
 DRA_OUTPUT="release-manager.out"
 PROJECT="fleet-server"
 TYPE=${1}
 BRANCH="${BUILDKITE_BRANCH}"
 
+make get-version
```

Comment:
This should be added to a VERSION env. var , as it was done in Jenkins 

https://github.com/elastic/fleet-server/blob/883cb46647979e234f70c88fd4aca1dc9f65800b/.ci/packaging.groovy#L66
Maybe like this ? 

```suggestion
VERSION=$(make get-version)
```